src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/brgemm_executor.cpp

```diff
@@ -0,0 +1,383 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "brgemm_executor.hpp"
+
+#include "dnnl_extension_utils.h"
+#include "utils/cpu_utils.hpp"
+#include <cpu/x64/cpu_isa_traits.hpp>
+#include <openvino/core/except.hpp>
+
+using namespace dnnl::impl::cpu::x64;
+using namespace dnnl::impl;
+using namespace dnnl::impl::cpu::x64::matmul;
+
+#define THROW_ERROR(...) OPENVINO_THROW("brgemm executor Init Failure '", __VA_ARGS__)
+namespace ov {
+namespace intel_cpu {
+
+brgemmExecutor::brgemmExecutor(size_t M,
```

Comment:
I would say it is rather kernel, than executor. Would propose to:
1. Rename `brgemmExecutor` -> `BrgemmKernel`
2. Move into kernels/x64 (as long as it is not specific to sdpa). Also would help to avoid usage of #if defined(OPENVINO_ARCH_X86_64) inside the code

Reply:
Sure, let me move to x64 and in this release. This kernel is for bf16bf16f32, should we name it to BrgemmKernel_bf16bf16fp32 or just BrgemmKernel ?

Reply:
Just `BrgemmKernel` is enough